plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/TestKafkaWithConfluentSchemaRegistryMinimalFunctionality.java

```diff
@@ -84,172 +89,141 @@ protected static QueryRunner createQueryRunner(TestingKafka testingKafka)
     }
 
     @Test
-    @Flaky(issue = "https://github.com/trinodb/trino/issues/6412", match = "Error registering Avro schema: .*")
     public void testBasicTopic()
-            throws Exception
     {
-        try (TestingKafka testingKafka = TestingKafka.createWithSchemaRegistry();
-                QueryRunner queryRunner = createQueryRunner(testingKafka)) {
-            String topic = "topic-basic-MixedCase-" + randomTableSuffix();
-            assertTopic(
-                    testingKafka, queryRunner, topic,
-                    format("SELECT col_1, col_2 FROM %s", toDoubleQuoted(topic)),
-                    format("SELECT col_1, col_2, col_3 FROM %s", toDoubleQuoted(topic)),
-                    false,
-                    schemaRegistryAwareProducer(testingKafka)
-                            .put(KEY_SERIALIZER_CLASS_CONFIG, LongSerializer.class.getName())
-                            .put(VALUE_SERIALIZER_CLASS_CONFIG, KafkaAvroSerializer.class.getName())
-                            .build());
-        }
+        String topic = "topic-basic-MixedCase-" + randomTableSuffix();
```

Comment:
Is rename required ?

Reply:
What do you mean? The `randomTableSuffix` was already there. And now when we reusing kafka schema registry service it is a good practice to use unique names.